app/templates/partials/guidance.html

```diff
@@ -0,0 +1,17 @@
+{% import 'macros/helpers.html' as helpers %}
+
+{% for item in guidance_content.content %}
+  {%- if item.title -%}
+    <h3 class="venus">{{item.title}}</h3>
```

Comment:
Probably best if they're not `h3` as you can't rely on there being a `h2` above this. This was raised by DAC so a good time to fix it.